diff --git a/src/config.cpp b/src/config.cpp
index ab69543..41d2543 100644
--- a/src/config.cpp
+++ b/src/config.cpp
@@ -4,6 +4,7 @@
 #include <cstdlib>
 #include <fstream>
 #include <iostream>
+#include <sstream>
 #include <map>
 #include <string>

@@ -15,6 +16,8 @@
 #include "puzzles.hpp"
 #include "paths.hpp"

+using namespace std;
+
 struct Parser {
     Config * cfg;
     std::map<std::string, int> color_order;
