From 898bae517a42c1bf088e1c7b2bd34cd1fb086d22 Mon Sep 17 00:00:00 2001
From: Alistair Francis <alistair@alistair23.me>
Date: Wed, 3 Nov 2021 22:23:46 +1000
Subject: [PATCH] Makefile: Remove -Werror

Signed-off-by: Alistair Francis <alistair@alistair23.me>
---
 Makefile | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/Makefile b/Makefile
index aa27ff2..3402007 100644
--- a/Makefile
+++ b/Makefile
@@ -8,11 +8,9 @@ objects = \
 	3rdparty/hmac_sha/hmac_sha2.o \
 	3rdparty/hmac_sha/sha2.o
 
-CHECKFLAGS = -Wall -Werror -Wuninitialized -Wundef
-
 DEPFLAGS = -Wp,-MMD,$(@D)/.$(@F).d,-MT,$@
 
-override CFLAGS := $(CHECKFLAGS) $(AM_CFLAGS) $(CFLAGS)
+override CFLAGS := $(AM_CFLAGS) $(CFLAGS)
 
 INSTALL = install
 prefix ?= /usr/local
@@ -24,7 +22,7 @@ progs = mmc
 
 # make C=1 to enable sparse
 ifdef C
-	check = sparse $(CHECKFLAGS)
+	check = sparse
 endif
 
 all: $(progs) manpages
-- 
2.31.1

