diff --git a/src/genie/gesture_parser.cpy b/src/genie/gesture_parser.cpy
index fa82b67..c94fd59 100644
--- a/src/genie/gesture_parser.cpy
+++ b/src/genie/gesture_parser.cpy
@@ -16,17 +16,18 @@ namespace genie:
   ;
 
   void run_command(string command):
+    debug "RUNNING COMMAND", command
+    string cmd = command + "&"
+    c_str := cmd.c_str()
+    _ := system(c_str)
+
     ui::TaskQueue::add_task([=]() {
       usleep(1e3 * 50)
-
-      debug "RUNNING COMMAND", command
-      string cmd = command
-      c_str := cmd.c_str()
-      _ := system(c_str)
-
       ui::MainLoop::reset_gestures()
     })
 
+
+
   input::SwipeGesture* build_swipe_gesture(GestureConfigData gcd):
     fb := framebuffer::get()
     fw, fh := fb->get_display_size()
