--- Makefile	2020-11-08 23:21:09.000000000 +1000
+++ Makefile.new	2021-02-10 12:25:00.078581500 +1000
@@ -60,8 +60,8 @@
   endif
   EXE=.exe
 else
-  CROSS_PREFIX=
-  EXE=
+  CROSS_PREFIX=$(CROSS_COMPILE)
+  EXE=.arm
 endif
 ifdef CONFIG_CLANG
   HOST_CC=clang
@@ -86,7 +86,7 @@
   endif
 else
   HOST_CC=gcc
-  CC=$(CROSS_PREFIX)gcc
+  CC=$(CROSS_PREFIX)gcc -march=armv7-a -mfpu=neon -mfloat-abi=hard -mcpu=cortex-a9 --sysroot=$(SDKTARGETSYSROOT)
   CFLAGS=-g -Wall -MMD -MF $(OBJDIR)/$(@F).d
   CFLAGS += -Wno-array-bounds -Wno-format-truncation
   ifdef CONFIG_LTO
